### PR TITLE
jetski data

### PR DESCRIPTION
**Pull request by SiggyF**

### CODE REVIEW

**Comment by SiggyF** [comment]
Please review this. I had some issues understanding the grouping of layers.

File: index.html [hunk 1]
```diff
@@ -12,16 +12,13 @@
...
-    <script src="http://cdn.pydata.org/bokeh/release/bokeh-widgets-0.12.4.js"></script>
-    <script src="http://cdn.pydata.org/bokeh/release/bokeh-api-0.12.4.js"></script>
-
-
     <title>sandmotor-viewer</title>
   </head>
   <body>
     <div id="app"></div>
     <!-- built files will be auto injected -->
+    <script defer src="http://cdn.pydata.org/bokeh/release/bokeh-0.12.4.js"></script>
```
Comment (CindyvdVries) [inline index.html:20]: Newest version 0.12.9, didn't work?

File: src/components/morphology-canvas.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                    [item.extent[0], item.extent[1]],
+                    [item.extent[2], item.extent[1]],
+                    [item.extent[2], item.extent[3]],
+                    [item.extent[0], item.extent[3]]
+                  ]
+                },
+                paint: {}
+              };
+              item.layer = layer;
+              console.log('layer', layer);
```
Comment (CindyvdVries) [inline src/components/morphology-canvas.js:65]: Delete console.log?

File: src/components/morphology-canvas.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+              item.source = source;
+              item.dest = dest;
+              item.ctxSrc = source.getContext('2d');
+              item.ctxDest = dest.getContext('2d');
+
+
+              item.id = 'morphology-canvas-' + i;
+
+              let layer = {
+                name: "Jetski-" + i,
```
Comment (CindyvdVries) [inline src/components/morphology-canvas.js:45]: Should be different. Bathymetry?

File: src/components/morphology-canvas.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+              item.video = video;
+              item.source = source;
+              item.dest = dest;
+              item.ctxSrc = source.getContext('2d');
+              item.ctxDest = dest.getContext('2d');
+
+
+              item.id = 'morphology-canvas-' + i;
+
+              let layer = {
```
Comment (CindyvdVries) [inline src/components/morphology-canvas.js:44]: Make a layer group structured as follows: 
{ name: "", 
type: "group",
active: "", 
data: [{ layer1 }, { layer2 }, { layer3 }]
}

**Review by CindyvdVries — APPROVED** [review]
Looking good!

**Pull request merged**